cpp/src/ray/config_internal.h

```diff
@@ -21,6 +21,8 @@
 namespace ray {
 namespace api {
 
+using ray::core::WorkerType;
```

Comment:
Should we avoid `using` in header files?

Reply:
This header is not exposed to users. So I think it's fine. we should avoid using in api headers.